src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -436,6 +436,11 @@ CreateDistributedTableConcurrently(Oid relationId, char *distributionColumnName,
 
 	char replicationModel = DecideDistTableReplicationModel(distributionMethod,
 															colocateWithTableName);
+	if (replicationModel != REPLICATION_MODEL_STREAMING)
+	{
+		ereport(ERROR, (errmsg("cannot create a replicated distributed "
```

Comment:
I think we don't use `replicated distributed table` terminology much.

Maybe: errmsg: Citus only allows concurrent table distribution for distributed tables when citus.shard_replication_factor > 1 
             errhint: table %s is colocated with %s which has citus.shard_replication_factor > 1

and, before giving the hint, we could assert for !IsColocateWithDefault/!IsColocateWithNone. In case these are true, we'd already give this error above with ShardReplicationFactor > 1